Makefile

```diff
@@ -85,6 +83,11 @@ end-to-end-test: shared
 	sudo docker build -f Dockerfile.with-coverage -t welder/web-with-coverage:latest .
 	sudo docker run -d --name web -p 3000:3000 --restart=always --network welder welder/web-with-coverage:latest
 
+	until curl http://localhost:4000/api/status | grep '"supported":true'; do \
```

Comment:
Might be better to use curl --connection-timeout here so that it can't get completely stuck.